### ISSUE TITLE
trouble with extract_area

### ISSUE THREAD

**Issue by iowatom** [libvips/php-vips#91]
I have been generating tiled images for web based virtual microscopy using extract_area with .mrxs files (~1 gbyte) as the source. I had been using vips 8.4.5 on Ubuntu.

Command used:
vips extract_area ImageName.mrxs[level=2] ImageName/tiles.dz 3744 4128 9056 7648

This is a screen shot of the image displayed using openseadragon:
<img width="144" alt="vips8 4 5" src="https://user-images.githubusercontent.com/8562847/64975233-1d3f3b80-d874-11e9-8675-e9112a19c531.png">

I recently upgraded to vips 8.8.2 using brew on a Mac running mojave.

Here is the result rendering the same .mrxs file with vips 8.8.2:
<img width="144" alt="vips8 8 2" src="https://user-images.githubusercontent.com/8562847/64975235-1d3f3b80-d874-11e9-8e62-d810b2619203.png">

**Comment by kleisauke** [comment]
Hello, I guess you installed vips with Homebrew? Homebrew [ships with Pixman 0.38.4](https://github.com/Homebrew/homebrew-core/blob/82495eedd0e6c274ac47d40f1feb1fa004f427b4/Formula/pixman.rb#L4) which contains a bug that causes the tiles to be incorrectly rendered for the scaled-down versions of the high-resolution image (level > 0).

This has already been fixed upstream, see:
https://gitlab.freedesktop.org/pixman/pixman/commit/8256c235d9b3854d039242356905eca854a890ba

But there's not yet a new version of Pixman available. For more information about this, see:
https://github.com/libvips/libvips/issues/1401

**Comment by iowatom** [comment]
I recently upgraded vips on my mac using homebrew ( to version 8.9.0).

There had been a problem with tile corruption when producing deep zoom (dzi) image files with the previous installers (8.8.2) created by a bug in Pixman. This appears to now be fixed.

However now there seems be be problem with the level option in openslide images.

The command line:
 vips dzsave image.mrxs[level=3] out_folder/tiles.dz

fails with the error:
   jpegload_source: no property named `level'

If I omit the the [level=3] option, it produces a very large image that is well formed.
I want to make smaller images.
Any idea how I might fix this?

**Comment by jcupitt** [comment]
Hello @iowatom, this is a bug in 8.9.0. It's fixed in 8.9.1, so just wait for the update.

**Comment by jcupitt** [comment]
In the meantime, you can use:

```
vips openslideload image.mrxs out_folder/tiles.dz --level 3
```
